src/main/java/Duke/Helper/DisplayMessages.java

```diff
@@ -0,0 +1,74 @@
+package Duke.Helper;
+
+import Duke.Duke;
+import Duke.Tasks.Task;
+
+import java.util.ArrayList;
+
+public class DisplayMessages {
```

Comment:
You might want to consider renaming the class, as plural is usually used to indicate arrays. In general, nouns are more suitable for classes while verbs are usually more suitable for interfaces.